tests/si_aggregator_test.py

```diff
@@ -78,5 +78,7 @@ def test_create_si_aggregated_json(self) -> None:
         with open(self._SI_AGGREGATOR_REPORTS_FILE) as json_file:
             aggregated_json_test = json.load(json_file)
         keys_tests = sorted([k for k in aggregated_json_test.keys()])
+        print(keys)
```

Comment:
Are these prints intentional?

Reply:
Nice catch!! Removed them in af11332db026a6ed160f06ec7597d6f57a5568be